### PR TITLE
Language selector + dutch translation

### PR DESCRIPTION
**Pull request by ianwijma**
Added Dutch translations.
Because the flags at the bottom where a bit messy, I have created a LanguageSelector which I placed on the login screen and replaced the one in the settings with the same LanguageSelector.

**Note: Maybe we should first merge either this one or #202 because of the overlapping changes.**

### CODE REVIEW

**Review by adityaruplaha — COMMENTED** [review]
Mostly looks good. A very minor nit though. :P

**Comment by ianwijma** [comment]
@adityaruplaha Please tell me what is your minor nit? :smile:

File: src/components/UI/LanguageSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <select 
+      onChange={(event) => handleLanguageChange(event.target.value)}
+      className={className}
+      value={currentLanguage}
+    >
+      {Object.keys(languageLabels).map(lang => renderOption(lang))}
+    </select>
+  )
+}
```
Comment (adityaruplaha) [inline src/components/UI/LanguageSelector.tsx:62]: Please add a trailing newline here. Nothing serious. :P

Comment (ianwijma) [reply src/components/UI/LanguageSelector.tsx]: Fixed! :)

Comment (adityaruplaha) [reply src/components/UI/LanguageSelector.tsx]: Thanks!

**Comment by flavioislima** [comment]
@ianwijma can you fix the conflicts and then we can merge :D

**Comment by ianwijma** [comment]
> @ianwijma can you fix the conflicts and then we can merge :D

Fixed =D

**Review by flavioislima — APPROVED** [review]
Nice 👍🏽

**Pull request merged**